### DISCUSSION TITLE
Working with react js

### DISCUSSION THREAD

**Discussion by rahimbtc** [spookylukey/django-paypal#251] [Q&A]
hello everyone,
This is not an issue but a question, is there a way to use django-paypal with reactjs (paypalreactform ...) ?

**Comment by newearthmartin** [comment]
Hey, 
I use django-paypal to process the IPNs but for the frontend I just use the new PayPal js framework, so I have some React components that call the PayPal js api in `componentDidMount()` and `componentDidUpdate()`.
